Formula/pmd-emacs.rb

```diff
@@ -0,0 +1,30 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class PmdEmacs < EmacsFormula
+  desc "Major mode for plugging PMD into Emacs"
+  homepage "https://github.com/pmd/pmd-emacs"
+  url "https://github.com/pmd/pmd-emacs/archive/0.8.tar.gz"
+  sha256 "d4677bf1039f2a25b0a029c593bfc1f442c8fb0807b17092ce422cd25f390f42"
+  head "https://github.com/pmd/pmd-emacs.git"
+
+  depends_on :emacs => "22.1"
+  depends_on "pmd"
+
+  def install
+    # install the documentation
+    doc.install "CHANGELOG", "LICENSE.txt", "readme.md"
```

Comment:
I initially expected it to do that, but without this line it didn't actually seem to install them. I'll try it again without this line and see if it actually is necessary.